### PR TITLE
[Rule tuning] Update rules based on docs review

### PR DESCRIPTION
**Pull request by brokensound77**
## Issues
None
related to doc [review](https://github.com/elastic/security-docs/pull/1283#pullrequestreview-824201789) from elastic/security-docs/pull/1283

## Summary
Update rules metadata

### CODE REVIEW

File: rules/integrations/azure/defense_evasion_firewall_policy_deletion.toml [hunk 1]
```diff
@@ -8,12 +8,12 @@ integration = "azure"
 author = ["Elastic"]
 description = """
 Identifies the deletion of a firewall policy in Azure. An adversary may delete a firewall policy in an attempt to evade
-defenses and/or to eliminate barriers in carrying out their initiative.
+defenses and/or to eliminate barriers go their objective.
```
Comment (w0rk3r) [inline rules/integrations/azure/defense_evasion_firewall_policy_deletion.toml:11]:
```suggestion
defenses and/or to eliminate barriers to their objective.
```
Typo?

File: rules/integrations/azure/defense_evasion_frontdoor_firewall_policy_deletion.toml [hunk 2]
```diff
@@ -8,12 +8,12 @@ integration = "azure"
 author = ["Austin Songer"]
 description = """
 Identifies the deletion of a Frontdoor Web Application Firewall (WAF) Policy in Azure. An adversary may delete a Frontdoor Web Application Firewall 
-(WAF) Policy in an attempt to evade defenses and/or to eliminate barriers in carrying out their initiative.
+(WAF) Policy in an attempt to evade defenses and/or to eliminate barriers go their objective.
```
Comment (w0rk3r) [inline rules/integrations/azure/defense_evasion_frontdoor_firewall_policy_deletion.toml:11]:
```suggestion
(WAF) Policy in an attempt to evade defenses and/or to eliminate barriers to their objective.
```
Typo?

File: rules/windows/credential_access_mimikatz_powershell_module.toml [hunk 3]
```diff
@@ -44,7 +44,7 @@ Atomic Red Team or through offensive/compromise assessments.
 - Modification of WDigest Security Provider
 
 ### Response and Remediation
-- Immediate response should be taken to review, investigate and potentially isolate activity to prevent further post-compromise
+- take immediate action to review, investigate and potentially isolate activity to prevent further post-compromise
```
Comment (w0rk3r) [inline rules/windows/credential_access_mimikatz_powershell_module.toml:47]:
```suggestion
- Take immediate action to review, investigate and potentially isolate activity to prevent further post-compromise
```
Change case

File: rules/windows/credential_access_suspicious_lsass_access_via_snapshot.toml [hunk 4]
```diff
@@ -9,7 +9,7 @@ min_stack_comments = "Cardinality field not added to threshold rule type until 7
 [rule]
 author = ["Elastic"]
 description = """
-Identifies suspicious access to an LSASS handle via PssCaptureSnapShot where two successive process access are performed
+Identifies suspicious access to an LSASS handle via PssCaptureo where two successive process access are performed
```
Comment (w0rk3r) [inline rules/windows/credential_access_suspicious_lsass_access_via_snapshot.toml:12]:
```suggestion
Identifies suspicious access to an LSASS handle via PssCaptureSnapShot where two successive process access are performed
```

Revert this change

File: rules/windows/lateral_movement_dns_server_overflow.toml [hunk 5]
```diff
@@ -31,16 +31,16 @@ also known as [SigRed](https://www.elastic.co/blog/detection-rules-for-sigred-vu
 - This specific rule is sourced from network log activity such as DNS or network level data. It's important to validate
 the source of the incoming traffic and determine if this activity has been observed previously within an environment.
 - Activity can be further investigated and validated by reviewing available corresponding Intrusion Detection Signatures (IDS) alerts associated with activity.
-- Further examination can be made by reviewing the `dns.question_type` network fieldset with a protocol analyzer, such as Zeek, Packetbeat, or Suricata, for `SIG` or `RRSIG` data.
+- Further examination can include review of the `dns.question_type` network fieldset with a protocol analyzer, such as Zeek, Packetbeat, or Suricata, for `SIG` or `RRSIG` data.
```
Comment (w0rk3r) [inline rules/windows/lateral_movement_dns_server_overflow.toml:34]:
```suggestion
- Further examination can include a review of the `dns.question_type` network fieldset with a protocol analyzer, such as Zeek, Packetbeat, or Suricata, for `SIG` or `RRSIG` data.
```

File: rules/windows/lateral_movement_scheduled_task_target.toml [hunk 6]
```diff
@@ -15,12 +15,13 @@ note = """## Triage and analysis
...
+[Scheduled tasks](https://docs.microsoft.com/en-us/windows/win32/taskschd/about-the-task-scheduler) are a great
+mechanism for persistence and program execution. These features can
 be used remotely for a variety of legitimate reasons, but at the same time used by malware and adversaries.
-When investigating scheduled tasks that have been set-up remotely, one of the first methods should be determining the
-original intent behind the configuration and verify if the activity is tied to benign behavior such as software installations or any kind
+When investigating scheduled tasks that were set up remotely, one of the first steps should be to determine the
+original intent behind the configuration and to verify if the activity is tied to benign behavior such as software installation or any kind
 of network administrator work. One objective for these alerts is to understand the configured action within the scheduled
-task, this is captured within the registry event data for this rule and can be base64 decoded to view the value.
+task This is captured within the registry event data for this rule and can be base64 decoded to view the value.
```
Comment (w0rk3r) [inline rules/windows/lateral_movement_scheduled_task_target.toml:24]:
```suggestion
task. This is captured within the registry event data for this rule and can be base64 decoded to view the value.
```

**Review by w0rk3r — APPROVED** [review]

File: rules/windows/credential_access_suspicious_lsass_access_via_snapshot.toml (hunk 4 above)
Comment (brokensound77) [reply rules/windows/credential_access_suspicious_lsass_access_via_snapshot.toml:12]: No clue what happened here - nice catch

**Review by Samirbous — APPROVED** [review]
